mdm/common/src/main/java/com/gentics/mesh/core/image/ImageManipulator.java

```diff
@@ -1,60 +1,70 @@
 package com.gentics.mesh.core.image;
 
-import java.awt.image.BufferedImage;
-import java.io.InputStream;
-import java.util.Map;
-
 import com.gentics.mesh.core.data.binary.HibBinary;
 import com.gentics.mesh.parameter.ImageManipulationParameters;
-
+import io.reactivex.Completable;
 import io.reactivex.Single;
 
+import java.awt.image.BufferedImage;
+import java.io.InputStream;
+import java.util.Map;
+
 /**
  * SPI provider interface for image manipulators.
  */
 public interface ImageManipulator {
 
-	/**
```

Comment:
Something happened to this file's indentation - it looks like unchanged code-wise.